### PR TITLE
[Refactor] Update all payloads to DataContracts, Removed Subtypes in AuditLogChange

### PR DESCRIPTION
**Pull request by velddev**
Might've missed some [DataContract]-s. Not sure if this AuditLogChange is better, but check it out.

### CODE REVIEW

File: Spectacles.NET.Types/AuditLogs/AuditLogEntry.cs [hunk 1]
```diff
@@ -1,52 +1,53 @@
...
+using System.Runtime.Serialization;
 
 namespace Spectacles.NET.Types
 {
 	/// <summary>
 	///     An Audit Log Entry
 	/// </summary>
-	public class AuditLogEntry
+	[DataContract]
+    public class AuditLogEntry
```
Comment (appellation) [inline Spectacles.NET.Types/AuditLogs/AuditLogEntry.cs]:
```suggestion
	public class AuditLogEntry
```

File: Spectacles.NET.Types/AuditLogs/AuditLogEntryInfo.cs [hunk 2]
```diff
@@ -1,58 +1,59 @@
...
+using System.Runtime.Serialization;
 
 namespace Spectacles.NET.Types
 {
 	/// <summary>
 	/// Additional Audit Log info for certain action types
 	/// </summary>
-	public class AuditLogEntryInfo
+	[DataContract]
+    public class AuditLogEntryInfo
```
Comment (appellation) [inline Spectacles.NET.Types/AuditLogs/AuditLogEntryInfo.cs]:
```suggestion
	public class AuditLogEntryInfo
```

File: Spectacles.NET.Types/Dispatch/SendableDispatch.cs [hunk 3]
```diff
@@ -1,22 +1,23 @@
...
+using System.Runtime.Serialization;
 
 namespace Spectacles.NET.Types
 {
 	/// <summary>
 	///     A Packet Sent from the Worker to the Gateway where only the GuildId is known and not the ShardCount.
 	/// </summary>
-	public class SendableDispatch
+	[DataContract]
+    public class SendableDispatch
```
Comment (appellation) [inline Spectacles.NET.Types/Dispatch/SendableDispatch.cs:9]:
```suggestion
	public class SendableDispatch
```

File: Spectacles.NET.Types/VoiceState/VoiceState.cs [hunk 4]
```diff
@@ -1,70 +1,71 @@
-using Newtonsoft.Json;
+using System.Runtime.Serialization;
 
 namespace Spectacles.NET.Types
 {
-	/// <summary>
+    /// <summary>
```
Comment (appellation) [inline Spectacles.NET.Types/VoiceState/VoiceState.cs]:
```suggestion
	/// <summary>
```

File: Spectacles.NET.Types/Webhook/Webhook.cs [hunk 5]
```diff
@@ -1,53 +1,55 @@
-using Newtonsoft.Json;
+using System.Runtime.Serialization;
 
 namespace Spectacles.NET.Types
 {
-	/// <summary>
+
+    /// <summary>
```
Comment (appellation) [inline Spectacles.NET.Types/Webhook/Webhook.cs]:
```suggestion
	/// <summary>
```

**Review by appellation — COMMENTED** [review]
Not sure how important indentation is to @DevYukine, but a lot of the files you've changed include changing indentation from tabs to 2 spaces. This introduces some inconsistency.

**Review by DevYukine — CHANGES_REQUESTED** [review]
What @appellation said, please change the indentation back to tabs, beside that lgtm

**Review by DevYukine — APPROVED** [review]
lgtm

**Pull request merged**